Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8
+FROM eclipse-temurin:17
```

Comment:
As per the [official openjdk dockerhub](https://hub.docker.com/_/openjdk) the openjdk docker images are deprecated, so this is a replacement using one of the suggested images that are maintained going forward.